pkg/api/v1/types.go

```diff
@@ -2048,7 +2048,7 @@ type NodeSpec struct {
 // DaemonEndpoint contains information about a single Daemon endpoint.
 type DaemonEndpoint struct {
 	// Port number of the given endpoint.
-	Port int32 `json:port`
+	Port int32 `json:"port"`
```

Comment:
Ugh.

@gmarek Is this API in use anywhere?

We may need to ask on kubernetes-dev whether this breaks any clients.